packages/subgraph/src/mapping.ts

```diff
@@ -86,23 +87,18 @@ export function handleTransfer(event: Transfer): void {
     return;
   }
 
+  // No need to update the underlyingAmount as the contract already consider
```

Comment:
This is interesting because the smart contract already handles the claimed underlying amount logic for us. So only updating the `claimedUnderlyingAmount` for a user on every token transfer of the VestERC20 token we keep up to date to the user vesting data.

Reply:
The token are transfered each blocktime? How the Transfer event is emitted?

Reply:
No, we are listening to this event: https://github.com/1Hive/vest/blob/b05fc5405e2ded8fec48a4751b1f49762a1e63ba/packages/hardhat/contracts/VestedERC20.sol#L207

The token transfers of the vested token (e.g. vHNY)